Cork/Views/Settings/Panes/General Pane.swift

```diff
@@ -58,12 +58,12 @@ struct GeneralPane: View
                 }
                 
                 Picker(selection: $caveatDisplayOptions) {
-                    Text("Full display")
+                    Text("Full")
                         .tag(PackageCaveatDisplay.full)
-                    Text("Minified display")
+                    Text("Minified")
                         .tag(PackageCaveatDisplay.mini)
                 } label: {
-                    Text("Package Caveats:")
+                    Text("Package caveats:")
```

Comment:
Consistency: only capitalise the first word of the phrase

Reply:
Agreed